components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/devportal-api.yaml

```diff
@@ -106,15 +106,15 @@ info:
     url: http://www.apache.org/licenses/LICENSE-2.0.html
   version: v2
 servers:
-- url: https://apis.wso2.com/api/am/devportal/v2
+  - url: https://apis.wso2.com/api/am/devportal/v2
```

Comment:
Let's do not format the full file?

Reply:
As discussed offline, we will send a PR to the master branch addressing all the formatting changes.